manifests/web.pp

```diff
@@ -151,7 +151,7 @@
   # zabbix frontend 5.x is not supported, among others, on stretch and xenial.
   # https://www.zabbix.com/documentation/current/manual/installation/frontend/frontend_on_debian
   if $facts['os']['name'] in ['ubuntu', 'debian'] and versioncmp($zabbix_version, '5') >= 0 {
-    if versioncmp($facts['os']['release']['major'], '16.04') == 0 or versioncmp($facts['os']['release']['major'], '9') == 0 {
+    if versioncmp($facts['os']['release']['major'], '9') == 0 {
```

Comment:
Should it also drop Debian 9, which we also consider unsupported in Vox Pupuli modules.

Reply:
I did like to drop it on another PR